staging/src/k8s.io/kubectl/testdata/set/multi-resource-yaml.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
       - name: mock-container
-        image: k8s.gcr.io/pause:3.7
+        image: registry.k8s.io/pause:3.8
```

Comment:
ok. we have another variant - 3.8 of `pause` here. It may be a good idea to align all to `3.8`